.github/workflows/bump-version.yml

```diff
@@ -12,26 +12,77 @@ on:
           - patch
           - minor
           - major
+  issue_comment:
+    types: [created]
 
 jobs:
   bump-version:
     name: 🔢 Bump Version
     runs-on: ubuntu-latest
+    if: >
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'issue_comment' && 
+       github.event.issue.pull_request &&
+       startsWith(github.event.comment.body, '/bump-version'))
     steps:
+      - name: 🔍 Parse Comment Command
+        id: parse-command
+        if: github.event_name == 'issue_comment'
+        run: |
+          comment="${{ github.event.comment.body }}"
+          if [[ $comment =~ ^/bump-version[[:space:]]+(patch|minor|major)$ ]]; then
+            version_type=$(echo "$comment" | awk '{print $2}')
+            echo "version_type=$version_type" >> $GITHUB_OUTPUT
+            echo "valid=true" >> $GITHUB_OUTPUT
+          elif [[ $comment =~ ^/bump-version[[:space:]]*$ ]]; then
+            echo "version_type=patch" >> $GITHUB_OUTPUT
+            echo "valid=true" >> $GITHUB_OUTPUT
+          else
+            echo "valid=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: ❌ Exit if Invalid Command
```

Comment:
Why should this workflow run with every comment? Why can't we just click "run" like we would to rebase and merge the PR? I've seen other workflows require to be ran manually on PRs before without having to add a comment. Thoughts?